src/FakeItEasy/Creation/CastleDynamicProxy/CastleDynamicProxyInterceptionValidator.cs

```diff
@@ -34,7 +34,7 @@ private static string GetReasonForWhyMethodCanNotBeIntercepted(MethodInfo method
                 var explicitImplementation = method.Name.Contains('.');
                 if (!explicitImplementation)
                 {
-                    return "Sealed methods can not be intercepted.";
+                    return "Method should be marked as virtual, abstract or part of an Interface in oreder to be intercepted.";

```

Comment:
I like the intent here, especially the idea of giving the user a way forward, rather than just complaining at them (and you found a way to avoid the "can not/cannot" debate, which is always fun).

I want to take a little time to trigger this error in context to see how it reads, but while I'm here, I'll make a few comments before I forget.

1. "order" is misspelled,
2. "Interface" is capitalized for some reason

But maybe don't jump right on this, as I'll probably have more to say after an experiment…